build_dev.sh

```diff
@@ -14,5 +14,14 @@ mkdir environments/dev/manifests
 jsonnet -J vendor -m environments/dev/manifests environments/dev/main.jsonnet | xargs -I{} sh -c 'cat {} | gojsontoyaml > {}.yaml' -- {}
 find environments/dev/manifests -type f ! -name '*.yaml' -delete
 
+rm -rf tests/manifests
+mkdir tests/manifests
+
 jsonnet -J vendor -m tests/manifests tests/main.jsonnet | xargs -I{} sh -c 'cat {} | gojsontoyaml > {}.yaml' -- {}
 find tests/manifests -type f ! -name '*.yaml' -delete
+
+rm -rf example/manifests
+mkdir example/manifests
+
+jsonnet -J vendor example/main.jsonnet | gojsontoyaml >example/manifests/observatorium-crd.yaml
```

Comment:
I think this should be in a follow up PR, as I don't want to change more than necessary in this PR, but `-crd.yaml` is misleading, as this is a custom resource instance not a custom resource definition. I think in a follow up we should just rename this to `observatorium.yaml`.